<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AUDForumDlg</class>
 <widget class="QDialog" name="AUDForumDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>679</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add|Update|Delete Forums</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>661</width>
     <height>441</height>
    </rect>
   </property>
   <property name="title">
    <string>Forum Settings</string>
   </property>
   <widget class="QGroupBox" name="groupBox_Existing_Forum">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>340</y>
      <width>551</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Existing Forum Names</string>
    </property>
    <widget class="QListWidget" name="listWidget_Existing_forums">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>20</y>
       <width>271</width>
       <height>71</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QCheckBox" name="checkBox_Private_Forum">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>80</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Private Forum</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_Allow_New_Thrad">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>100</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Allow only Admins to Start a New Thread</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>220</y>
      <width>551</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="Forum_Type_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>         Select Group</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_Select_Group_AUD">
       <property name="placeholderText">
        <string>Select Group</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Participant EMail</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Retrive_Email_AUD">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>220</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Retrieve EMail</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>551</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>         Forum Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_Forum_name">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>551</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="Forum_Type">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>        Forum Type</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_Forum_Type">
       <property name="placeholderText">
        <string>Forum Type</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>130</y>
      <width>97</width>
      <height>16</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>         Description</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_Desc_AUD_Forum">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>131</y>
      <width>271</width>
      <height>71</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_Participant_Email">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>250</y>
      <width>271</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_AddNew">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>460</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Add New</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Add_AUD_Forum">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>460</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Delete_AUD_Forum">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>461</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Cancel_AUD_Forum">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>460</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
